src/shared/api/server/summary.ts

```diff
@@ -56,11 +57,43 @@ export async function GET(req: NextRequest): Promise<NextResponse<SummaryRespons
 
   const summary = results[0];
   if (!summary) {
-    return NextResponse.json(
-      { error: `No summary found for ${baseAssetSymbol}/${quoteAssetSymbol}` },
-      { status: 400 },
-    );
+    return NextResponse.json({ window: durationWindow, noData: true });
```

Comment:
Because it's so common to have no summary data, created a 200 success with another payload